src/Microsoft.CodeQuality.Analyzers/UnitTests/Maintainability/UseNameOfInPlaceOfStringTests.cs

```diff
@@ -139,9 +141,9 @@ class C
 {
     void M(int x)
     {
-        throw new ArgumentNullException(""test"", ""test2"", ""test3"");
+        throw new ArgumentNullException(""test"", ""test2"");
```

Comment:
📝 We should make sure this early return line of the analyzer is still covered by tests:

https://github.com/dotnet/roslyn-analyzers/blob/1fa5f5c1167500b93bbebc301f240edde60effce/src/Microsoft.CodeQuality.Analyzers/Core/Maintainability/UseNameofInPlaceOfString.cs#L64-L67

Reply:
No test is reaching this point. But keeping the invalid syntax and adding an expected compiler error still doesn't reach this point. I am happy to work on adding coverage for this condition but it might be better to have a separate PR, don't you think?

Reply:
Ticket #3115